README.md

```diff
@@ -659,6 +660,19 @@ Calculates a boundary box and centers its children accordingly.
 </Center>
 ```
 
+#### Preload
+
+Precompiles the scene using [gl.compile](https://threejs.org/docs/index.html#api/en/renderers/WebGLRenderer.compile), this makes sure that your scene is responsive from the get go. If you have async models you must drop it into the same suspense boundary *in concurrent mode*. If you don't then simply put it into the canvas. By default gl.compile will only preload visible objects, if you supply the `all` prop, it will circumvent that.
```

Comment:
So does this only work in concurrent mode?

Reply:
And if I have multiple models, do I need to put it in each suspense block?

Reply:
nope. works everywhere.

```jsx
<Canvas>
  <mesh>
    <boxBufferGeometry />
   </mesh>
   <Preload />
 </Canvas>
```

now it will precompile the material and block render until it's done. this is common in threejs, people use gl.compile all the time. we just never had a declarative means to do it but you could have always just called it.

threejs people WILL face trouble with async stuff though because they have no means to manage it. this is the great part, preload will work here, too. but it must be concurrent, or else contents dont fire in lockstep - this now is a react thing and wouldn't be the place to explain. react has three renderers: legacy, blocking, cm. legacy works differently in suspense.

Reply:
added more explanations